common/k8sinstall/util.go

```diff
@@ -220,3 +220,31 @@ func ScaleZfsControllerViaHelm(expected_replica int32) (int32, error) {
 
 	return orig_replicas, nil
 }
+
+// SetRdmaViaHelm enable and disable RDMA
+func SetRdmaViaHelm(enableRdma bool, iface string) error {
+	e2eCfg := e2e_config.GetConfig()
+	values := map[string]interface{}{
+		"io_engine.target.nvmf.rdma.enabled": enableRdma,
+		"io_engine.target.nvmf.rdma.iface":   iface,
```

Comment:
Btw, is it that or this?

```suggestion
		"io_engine.target.nvmf.iface":   iface,
```

Reply:
> Btw, is it that or this?

It's "io_engine.target.nvmf.iface"  in [PR](https://github.com/openebs/mayastor-extensions/pull/554/files)
I will update my PR

Reply:
[fixed](https://github.com/openebs/openebs-e2e/compare/0115fd0c8f576da8b5a62567a18db77c895fbc8e..3a52152bc3aefc830bd1b1a262d46dab0e255e4c)